### PR TITLE
Categorize java management libraries under "Resource Management"

### PR DESCRIPTION
**Pull request by nickzhums**
After the structure update, some management packages appear under each service while some end up in "Others". We would like to make it consistent and since management SDK customers usually use a couple different services together for resource management, we propose to create a "Resource Management" category and put all packages under same category

### CODE REVIEW

**Review by weidongxu-microsoft — APPROVED** [review]

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -534,11 +534,11 @@
...
-"azure-resourcemanager-parent","com.azure.resourcemanager","","2.0.0-beta.3","azure-resourcemanager-parent","","NA","NA","NA","","",""
-"azure-resourcemanager-privatedns","com.azure.resourcemanager","","2.0.0-beta.5","azure-resourcemanager-privatedns","","NA","","","","",""
-"azure-resourcemanager-servicebus","com.azure.resourcemanager","","2.0.0-beta.5","azure-resourcemanager-servicebus","","NA","","","","",""
+"azure-resourcemanager","com.azure.resourcemanager","2.0.0","2.0.0-beta.4","azure-resourcemanager","Resource Management","NA","","","","",""
+"azure-resourcemanager-cdn","com.azure.resourcemanager","","2.0.0-beta.5","azure-resourcemanager-cdn","Resource Management","NA","","","","",""
+"azure-resourcemanager-parent","com.azure.resourcemanager","","2.0.0-beta.3","azure-resourcemanager-parent","Resource Management","NA","NA","NA","","",""
+"azure-resourcemanager-privatedns","com.azure.resourcemanager","","2.0.0-beta.5","azure-resourcemanager-privatedns","Resource Management","NA","","","","",""
+"azure-resourcemanager-servicebus","com.azure.resourcemanager","","2.0.0-beta.5","azure-resourcemanager-servicebus","Resource Management","NA","","","","",""
```
Comment (ChenTanyi) [inline _data/releases/latest/java-packages.csv:541]: This should align with above, `azure-resourcemanager` preview version should be `2.0.0-beta.5`, and hide `azure-resourcemanager-parent`

**Review by ChenTanyi — APPROVED** [review]

File: _data/releases/latest/java-packages.csv (hunk 1 above)
Comment (nickzhums) [reply _data/releases/latest/java-packages.csv:541]: i think the version will be updated automatically

**Review by sima-zhu — APPROVED** [review]
Looks good from github io perspective

File: _data/releases/latest/java-packages.csv [hunk 2]
```diff
@@ -39,25 +39,25 @@
...
-"azure-resourcemanager-redis","com.azure.resourcemanager","","2.0.0-beta.5","Resource Management - Redis","Redis","resourcemanager","","","mgmt","",""
-"azure-resourcemanager-resources","com.azure.resourcemanager","2.0.0","","Resource Management - Resources","Resources","resourcemanager","","","mgmt","",""
-"azure-resourcemanager-appplatform","com.azure.resourcemanager","","2.0.0-beta.5","Resource Management - Spring Cloud","Spring Cloud","resourcemanager","","","mgmt","",""
-"azure-resourcemanager-sql","com.azure.resourcemanager","","2.0.0-beta.5","Resource Management - SQL","SQL","resourcemanager","","","mgmt","",""
-"azure-resourcemanager-storage","com.azure.resourcemanager","2.0.0","","Resource Management - Storage","Storage","resourcemanager","","","mgmt","",""
-"azure-resourcemanager-trafficmanager","com.azure.resourcemanager","","2.0.0-beta.5","Resource Management - Traffic Manager","Traffic Manager","resourcemanager","","","mgmt","",""
+"azure-resourcemanager-appservice","com.azure.resourcemanager","2.0.0","","Resource Management - App Service","Resource Management","resourcemanager","","","mgmt","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:42]: Is this what we really want? Don't we want these to be grouped with the service they apply to? If this is the pattern we want we should apply the same pattern to the other languages as well.

Comment (nickzhums) [reply _data/releases/latest/java-packages.csv:42]: Hi Wes, sorry if i didn't clarify, we've done quite a few customer interviews, and found that most management SDK customers will use a variety of services together, for example, customers who use compute usually also uses resources and network. By grouping them under one category, the customers can avoid looking for packages in different services back and forth.

If you have better visualization suggestions, i'm all ears

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:42]: I don't have a strong opinion I just wanted to make sure you understand what this means. I do think that if you do it for java we should do it for .NET and python as well.

Comment (nickzhums) [reply _data/releases/latest/java-packages.csv:42]: @weshaggard What do you think about keeping the current structure, but adds a second-level dropdown like "Client Packages" and "Management Packages". The concern I have is sometimes when there is both data-plane and management-plane SDKs under same service, the customer doesn't really know which SDK suits their purpose. Let me know what you think about this

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:42]: I'm not sure I fully understand can you give me an example where people have both together but don't have the context of which ones are client vs mgmt?

Comment (nickzhums) [reply _data/releases/latest/java-packages.csv:42]: Take Storage for example: https://azure.github.io/azure-sdk-for-java/storage.html

You can see there is a bunch of SDKs on this page. There is no description other than the package name itself. The customer would have to guess which SDK suits their scenarios. 

From past surveys we also observe customers have no idea about whether the SDK is a data-plane or management-plane.

I think a second level dropdown can help differentiate the packages, this also is consistent with our current MS Docs structure approach.

Another option is to add some text under each package name to give clear message to customer

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:42]: Yes I agree that page is less then ideal but we also don't have any great place to pull extra information for each of these packages. I personally don't think separating the client vs mgmt libraries is going to be that helpful as most folks don't think of the libraries in that way. We could consider pulling in the service level readme, https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/README.md but even that I don't think exactly matches. Do keep in mind that our github.io site is mostly for testing of docs and less for direct customer consumption, as the customers should mostly be looking at docs.microsoft.com. However with that said if you feel there are some improvements that are fairly cheap and help some scenarios for folks consuming those docs please file an issue at https://github.com/Azure/azure-sdk-tools/. We do have one issue https://github.com/Azure/azure-sdk-tools/issues/1159 about trying to collapse the version lists which is growing pretty long.

**Comment by nickzhums** [comment]
per offline conversation with Wes, we've decided to keep the current structure but add a description text under each package

**Pull request closed without merging**